xlators/mgmt/glusterd/src/glusterd-scrub-svc.c

```diff
@@ -113,15 +113,25 @@ glusterd_scrubsvc_manager(glusterd_svc_t *svc, void *data, int flags)
 int
 glusterd_scrubsvc_start(glusterd_svc_t *svc, int flags)
 {
+    int i = 0;
     int ret = -1;
     dict_t *cmdict = NULL;
+    char key[16] = {0};
+    char *options[] = {svc->name, "--process-name", NULL};
 
     cmdict = dict_new();
     if (!cmdict) {
         gf_smsg(THIS->name, GF_LOG_ERROR, errno, GD_MSG_DICT_CREATE_FAIL, NULL);
         goto error_return;
     }
 
+    for (i = 0; options[i]; i++) {
+        ret = snprintf(key, sizeof(key), "arg%d", i);
+        ret = dict_set_strn(cmdict, key, ret, options[i]);
+        if (ret)
+            goto dealloc_dict;
+    }
+
```

Comment:
Both these functions are exactly same. Do you see any reason for them to be separate functions?

Reply:
Now that I think..yes! Both the functions are duplicates. Will make the necessary changes.

Reply:
Also, if we are removing duplicates, glusterd_quotadsvc_start is also similar, the only difference being a certain cmdline args. I think even that should be handled, as we are anyway removing duplicates.

Reply:
glusterd seems to have lot of duplicate code. If you want to send another patch removing more duplicate code, I will be happy to review.